docs/content/tutorials/oidc-custom-configuration.md

```diff
@@ -0,0 +1,199 @@
+2. Attach a `Volume` and `VolumeMount` to our deployment of the F5 NGINX Ingress Controller
+
+This setup will allow our custom configuration in our ConfigMap to override the contents of the default `oidc.conf` file.
+
+## Step 1 - Creating the ConfigMap
+
+Run the below command to generate a ConfgMap with the contents of the `oidc.conf` file.
+**NOTE** The ConfgMap must be deployed in the same `namespace` as the F5 NGINX Ingress Controller.
+```
+kubectl create configmap oidc-config-map --from-literal=oidc.conf="$(curl -k https://raw.githubusercontent.com/nginxinc/kubernetes-ingress/v3.1.1/internal/configs/oidc/oidc.conf)"
+```
+
+Use the `kubectl describe` command to confirm the contents of the ConfigMap are correct
+
+```
+kubectl describe configmap oidc-config-map
+```
+
+```
+Name:         oidc-config-map
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+oidc.conf:
+----
+    # Advanced configuration START
+    set $internal_error_message "NGINX / OpenID Connect login failure\n";
+    set $pkce_id "";
+    # resolver 8.8.8.8; # For DNS lookup of IdP endpoints;
+    subrequest_output_buffer_size 32k; # To fit a complete tokenset response
+    gunzip on; # Decompress IdP responses if necessary
+    # Advanced configuration END
+
+    ...
+    # Rest of config ammended
+```
+
+## Step 2 - Customising the default configuration
+Once the contents of the `oidc.conf` file has been added to the ConfigMap, you are free to customise the contents of this ConfigMap.
+In this example, we will add a comment to the top of the file to demonstrate it is overwriting the default contents.
+This comment will be `# >> Custom Comment for my OIDC file <<`
+
+```
+kubectl edit configmap oidc-config-map
+```
+
+Add the custom content:
+```
+# Please edit the object below. Lines beginning with a '#' will be ignored,
+# and an empty file will abort the edit. If an error occurs while saving this file will be
+# reopened with the relevant failures.
+#
+apiVersion: v1
+data:
+  oidc.conf: |2-
+        # >> Custom Comment for my OIDC file <<
+        # Advanced configuration START
+        set $internal_error_message "NGINX / OpenID Connect login failure\n";
+        set $pkce_id "";
+        # resolver 8.8.8.8; # For DNS lookup of IdP endpoints;
+        subrequest_output_buffer_size 32k; # To fit a complete tokenset response
+        gunzip on; # Decompress IdP responses if necessary
+        # Advanced configuration END
+
+        ...
+        # Rest of config ammended
+```
+> **IMPORTANT**
+>
+> In Step 3 we will deploy/update an Ingress Controller that will use this ConfigMap. Any changes made to this ConfigMap must be made **before** deploying/updating the Ingress Controller. If an update is applied to the ConfigMap after the Ingress Controller is deployed, it will not get applied. Applying any updates to the data in this ConfigMap will require the Ingress Controller to be re-deployed.
```

Comment:
```suggestion
> In Step 3 an NGINX Ingress Controller will be deployed/updated that will use this ConfigMap. Any changes made to this ConfigMap must be made **before** deploying/updating the NGINX Ingress Controller. If an update is applied to the ConfigMap after the NGINX Ingress Controller is deployed, it will not get applied. Applying any updates to the data in this ConfigMap will require the NGINX Ingress Controller to be re-deployed.
```
The continuous use of the noun makes the full product name a bit painful here, but adding this suggestion for consistency.